### ISSUE TITLE
Missing "bootstrap" directory in box.json

### ISSUE THREAD

**Issue by bdimitrov** [phalcon/phalcon-devtools#943]
```
PHP Warning:  require(phar:///private/tmp/phalcon-devtools/phalcon.phar/bootstrap/autoload.php): 
failed to open stream: phar error: "bootstrap/autoload.php" is not a file in phar "/private/tmp/phalcon-devtools/phalcon.phar" 
in phar:///private/tmp/phalcon-devtools/phalcon.phar/phalcon.php on line 40
PHP Stack trace:
PHP   1. {main}() /private/tmp/phalcon-devtools/phalcon.phar:0
PHP   2. require() /private/tmp/phalcon-devtools/phalcon.phar:10
```

**Comment by clholzin** [comment]
same issue

**Comment by sergeyklay** [comment]
I'll fix it asap

**Comment by sergeyklay** [comment]
Fixed in the `3.0.x` branch.
